modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -306,6 +306,42 @@ async fn query_sboms_by_ingested_time(ctx: &TrustifyContext) -> Result<(), anyho
     Ok(())
 }
 
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn query_sboms_by_label(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let query = async |q| {
+        let app = caller(ctx).await.unwrap();
+        let uri = format!("/api/v2/sbom?q={}", encode(q));
+        let req = TestRequest::get().uri(&uri).to_request();
+        let response: Value = app.call_and_read_body_json(req).await;
+        assert_eq!(1, response["total"], "for {q}");
+    };
+    ctx.ingest_document_as(
+        "zookeeper-3.9.2-cyclonedx.json",
+        Format::CycloneDX,
+        [
+            ("type", "cyclonedx"),
+            ("source", "test"),
+            ("importer", "none"),
+            ("file", "zoo.json"),
+            ("datasetFile", "none"),
+        ],
+    )
+    .await?;
+
+    query("type!=spdx").await;
```

Comment:
It looks to me as if there's no indication that this is a label. What would be the case if there's a column `namespace` from PURLs, and the user applies a label `namespace`?

Reply:
Dunno. What do you think should be the case?

Reply:
I would want to query for labels explicitly.

Reply:
Be specific. Propose your desired syntax for querying labels.

Reply:
I don't really know the current syntax. All I would like to to is to say "all items with label `foo` equals/contains/… `bar`. Not conflicting with any other column named `foo`.

Reply:
> I don't really know the current syntax. 

I know. Suggest a new one then. Be constructive.

> All I would like to to is to say "all items with label `foo` equals/contains/… `bar`. Not conflicting with any other column named `foo`.

I understand. This PR doesn't provide that capability. Which is why it's only marginally related to #491 -- it does not fully address it and therefore won't close it.

Reply:
That's easy enough, but it requires [one of the commits](https://github.com/trustification/trustify/pull/1534/commits/f600eca2ea0d3cc6a920f8eea715cfab03d854fc) in #1534, so if you're gonna reject that one, I'll need to pull it out into its own PR.

Reply:
Done